examples/src/java/com/twitter/heron/examples/api/AckingTopology.java

```diff
@@ -103,18 +108,17 @@ public void close() {
     }
 
     public void nextTuple() {
-      // We explicitly slow down the spout to avoid the stream mgr to be the bottleneck
-      Utils.sleep(1);
-
       final String word = words[rand.nextInt(words.length)];
 
-      // To enable acking, we need to emit tuple with MessageId, which is an object
+      // To enable acking, we need to emit each tuple with a MessageId, which is an Object
```

Comment:
Can you add the following line as well
Technically each new message that we emit should be annottated with a unique message id. This will allow the spout to keep track of which messages should be acked back to the producer/retried when appropriate ack/fail happens. However for the sake of simplicitity we are tagging all tuples in this example to have the same message id.